src/main/java/com/znsio/rpi/properties/Config.java

```diff
@@ -0,0 +1,43 @@
+package com.znsio.rpi.properties;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+
+import javax.validation.constraints.NotNull;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class Config {
+    public static final String TARGET_ENVIRONMENT = "TARGET_ENVIRONMENT";
+    public static final String PLATFORM = "PLATFORM";
+    public static final String APP_NAME = "APP_NAME";
+    public static final String IS_VISUAL = "IS_VISUAL";
+    public static final String IS_MOBILAB = "IS_MOBILAB";
+    public static final String BROWSER = "BROWSER";
+    public static final String APP_PACKAGE_NAME = "APP_PACKAGE_NAME";
```

Comment:
Along with this, would it be imp to know if it is android or iOS?

Reply:
That will come from the PLATFORM value: web, android, ios